page.codeberg.JakobDev.jdMinecraftLauncher.yaml

```diff
@@ -11,8 +11,8 @@ sdk-extensions:
   - org.freedesktop.Sdk.Extension.openjdk17
   - org.freedesktop.Sdk.Extension.openjdk8
 finish-args:
-  - --socket=fallback-x11
   - --socket=wayland
+  - --socket=x11
```

Comment:
Why was this changed?

Reply:
Oh hmm I just remembered https://github.com/flathub-infra/flatpak-builder-lint/pull/310

Reply:
Yea I approved it already